crates/warg/tests/package.rs

```diff
@@ -1,81 +1,105 @@
-use std::fs::{self, DirEntry};
-
-use hashbrown::HashMap;
+use anyhow::{anyhow, Context, Result};
 use pretty_assertions::assert_eq;
-use semver::Version;
 use serde::{Deserialize, Serialize};
-use serde_json;
+use std::{
+    fs::{self, DirEntry},
+    path::Path,
+};
 use warg_protocol::{
     hash,
-    package::{self, validate::ValidationState},
+    package::{self, validate::Validator},
     protobuf, signing, Envelope,
 };
 
 #[test]
-fn test_package_logs() {
```

Comment:
I get using `?` early returns in main code, but in a test I usually like that `unwrap` or `expect` will bail where it failed and give me a convenient trace.